README.md

```diff
@@ -51,18 +51,18 @@ daily:20.10                 devel,groovy      20200721         Ubuntu 20.10
 
 ## Launch a fresh instance of the current Ubuntu LTS
 ```
-$ multipass launch ubuntu
+$ multipass launch 20.04
 Launching dancing-chipmunk...
-Downloading Ubuntu 18.04 LTS..........
+Downloading Ubuntu 20.04 LTS..........
 Launched: dancing chipmunk
 ```
 
 ## Check out the running instances
 ```
 $ multipass list
 Name                    State             IPv4             Release
-dancing-chipmunk        RUNNING           10.125.174.247   Ubuntu 18.04 LTS
-live-naiad              RUNNING           10.125.174.243   Ubuntu 18.04 LTS
+dancing-chipmunk        RUNNING           10.125.174.247   Ubuntu 20.04 LTS
+live-naiad              RUNNING           10.125.174.243   Ubuntu 20.04 LTS
```

Comment:
Just do a find/replace for `18.04`, you've missed a few instances in the `info` output and `lsb_release` output, for example.